src/sentry/incidents/apps.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import
+
+from django.apps import AppConfig
+
+
+class IncidentsAppConfig(AppConfig):
+    name = "sentry.incidents"
+
+    def ready(self):
+        from . import receivers  # noqa
```

Comment:
@wedamija Ah dang, I just realized that this is going to be broken since something (not covered by tests) will expect `sentry.incidents.receivers`.